moviesnetwork/moviesdata/data.py

```diff
@@ -0,0 +1,13 @@
+class Actor(object):
+
+    def __init__(self, actor_id, name):
+        self.actor_id = actor_id
+        self.name = name
```

Comment:
We could create the field `movies` as an empty list here. It will be better to handle it later. Instead of checking for `None` before iterate, we will have an empty list instead.